src/views/product-integrations-landing/index.tsx

```diff
@@ -1,50 +1,236 @@
+						/>
+					)}
+
+					<h5>Component</h5>
+					{sortedComponents.map((component, index) => {
+						return (
+							<FacetCheckbox
+								key={component.id}
+								label={component.plural_name}
+								matching={component.occurances}
+								isChecked={componentCheckedArray[index]}
+								onChange={(e) =>
+									setComponentCheckedArray(
+										componentCheckedArray.map((v, i) => {
+											return i === index ? !v : v
+										})
+									)
+								}
+							/>
+						)
+					})}
+
+					{atLeastOneFacetSelected && (
+						<Button
+							title="Reset filters"
+							size="small"
+							onClick={(e) => {
+								setOfficialChecked(false)
+								setPartnerChecked(false)
+								setCommunityChecked(false)
+								setComponentCheckedArray(
+									componentCheckedArray.map((v, i) => {
+										return false
+									})
+								)
+							}}
+						/>
+					)}
+				</div>
+			</div>
+		</>
+	)
+}
+
+// Custom sidebar
+const IntegrationsSearchSidebar = (props: SidebarProps) => {
+	const product = useCurrentProduct()
+	return (
+		<>
+			{/* Desktop sidebar — render search facets */}
+			<div className="g-hide-with-mobile-menu">
+				<SidebarNavList>
+					<SidebarBackToLink
+						text={`${product.name} Home`}
+						href={`/${product.slug}`}
+					/>
+					<SidebarNavMenuItem item={{ divider: true }} />
+				</SidebarNavList>
+
+				<SearchFacets />
+			</div>
+			{/* Mobile sidebar - render sidebar */}
+			<div className="g-show-with-mobile-menu">
+				<Sidebar {...props} />
+			</div>
+		</>
+	)
+}
+
+const IntegrationsSearchDrawer = () => {
+	const { dialogOpen, setDialogOpen } = useMobileDrawerContext()
+	return (
+		<Dialog
+			isOpen={dialogOpen}
+			label="Integrations filters"
+			// eslint-disable-next-line @typescript-eslint/no-empty-function
+			onDismiss={() => {}}
+			variant="bottom"
+		>
+			<button
+				className={s.exitIcon}
+				onClick={() => setDialogOpen(false)}
+				aria-label="Cancel"
+				type="button"
+			>
+				<IconX16 />
+			</button>
+			<SearchFacets />
+		</Dialog>
+	)
+}
+
 export default function ProductIntegrationsLanding({
 	product,
 	integrations,
 }: ViewProps) {
+	const breadcrumbLinks = [
+		{
+			title: 'Developer',
+			url: '/',
+			isCurrentPage: false,
+		},
+		{
+			title: product.name,
+			url: `/${product.slug}`,
+			isCurrentPage: false,
+		},
+		{
+			title: 'Integrations',
+			url: `/${product.slug}/integrations`,
+			isCurrentPage: true,
+		},
+	]
+	const sidebarNavDataLevels = [
+		generateTopLevelSidebarNavData(product.name),
+		generateProductLandingSidebarNavData(product),
+		{
+			backToLinkProps: {
+				text: `${product.name} Home`,
+				href: `/${product.slug}`,
+			},
+			levelButtonProps: {
+				levelUpButtonText: `${product.name} Home`,
+				levelDownButtonText: 'Previous',
+			},
+			// Ensure menuItems is so that the AlternateSidebar is rendered
+			menuItems: undefined,
+			showFilterInput: false,
+			title: `${product.name} Integrations`,
+		},
```

Comment:
Can you abstract this to an exported function?  We'll want to add this level to the specific integration page nested below.

Reply:
Actually, let's worry about that later; I'll add a task.

I want to wait to see what design shakes out first before polishing out the sidebar here...